tests/default/DbaInferenceTest.php

```diff
@@ -19,6 +19,8 @@ public function dataFileAsserts(): iterable
 
         if ('pdo-pgsql' === getenv('DBA_REFLECTOR')) {
             yield from $this->gatherAssertTypes(__DIR__.'/data/pdo-pgsql.php');
+        } elseif ('dibi-mysql' === getenv('DBA_REFLECTOR')) {
+            yield from $this->gatherAssertTypes(__DIR__.'/data/dibi-mysql.php');
```

Comment:
I think you can move the ` yield from $this->gatherAssertTypes(__DIR__.'/data/dibi-mysql.php');` to the bottom of this method.

dibi will re-use the pdo-pgsql/mysqli etc reflector.. we don't have a `dibi-mysql` reflector. you can drop the if condition

Reply:
the PR has it afaik https://github.com/jakubvojacek/phpstan-dba/blob/dibi-phase-two/tests/ReflectorFactory.php#L70